test/python/unittest/API/APITest.py

```diff
@@ -1682,9 +1683,11 @@ def makeAvailableChunksRoundTrip(self, ext):
 
         read = io.Series(
             name,
-            io.Access_Type.read_only
+            io.Access_Type.read_only,
+            options="{\"defer_iteration_parsing\": true}"
```

Comment:
Probably not worth here, but for all larger JSON options in Python (e.g. `jsonConfig` above) you can create a pretty Python dictionary and make it JSON like this:
```py
import json

options = {"defer_iteration_parsing": True}
json_str = json.dumps(options)
```

e.g.
```suggestion
            options=json.dumps({"defer_iteration_parsing": True})
```

Simple for here:
```suggestion
            options='{"defer_iteration_parsing": true}'
```

Reply:
Ah, good to know. I'll leave it at this for now, then.